mintpy/prep_fringe.py

```diff
@@ -341,7 +341,7 @@ def prepare_stack(outfile, unw_file, metadata, processor, baseline_dir=None, box
     print('number of interferograms:', num_pair)
 
     # get list of *.unw.conncomp file
-    cc_files = [f'{x}.conncomp' for x in unw_files]
+    cc_files = [x.replace('.unw', '.conncomp') for x in unw_files]
```

Comment:
Given the default output names are as below, this modification does not make sense to me. @bbuzz31 Could you explain?

```bash
20150921_20200215.unw
20150921_20200215.unw.conncomp
```